python/mmSolver/utils/reproject.py

```diff
@@ -82,22 +82,35 @@ def create_reprojection_on_camera(cam_tfm, cam_shp):
     maya.cmds.connectAttr(cam_tfm + '.worldMatrix', node + '.cameraWorldMatrix')
     maya.cmds.connectAttr(cam_shp + '.focalLength', node + '.focalLength')
     maya.cmds.connectAttr(cam_shp + '.cameraAperture', node + '.cameraAperture')
-    maya.cmds.connectAttr(cam_shp + '.filmOffset', node + '.filmOffset')
     maya.cmds.connectAttr(cam_shp + '.filmFit', node + '.filmFit')
     maya.cmds.connectAttr(cam_shp + '.nearClipPlane', node + '.nearClipPlane')
     maya.cmds.connectAttr(cam_shp + '.farClipPlane', node + '.farClipPlane')
     maya.cmds.connectAttr(cam_shp + '.cameraScale', node + '.cameraScale')
 
     # Connect render settings attributes
     resolution_factor = 10000.0
-    mult_node = maya.cmds.createNode('multiplyDivide')
+    mult_node = maya.cmds.createNode('multiplyDivide', name='main_multiplyDivide1')
     maya.cmds.setAttr(mult_node + '.input2X', resolution_factor)
     maya.cmds.setAttr(mult_node + '.input2Y', resolution_factor)
     maya.cmds.connectAttr(cam_shp + '.horizontalFilmAperture', mult_node + '.input1X')
     maya.cmds.connectAttr(cam_shp + '.verticalFilmAperture', mult_node + '.input1Y')
     maya.cmds.connectAttr(mult_node + '.outputX', node + '.imageWidth')
     maya.cmds.connectAttr(mult_node + '.outputY', node + '.imageHeight')
 
+    # create 2d offset setup
+    offset_plus_minus_node = maya.cmds.createNode('plusMinusAverage', name='offset_plusMinusAverage1')
+    maya.cmds.connectAttr(cam_shp + '.filmOffset', offset_plus_minus_node + '.input2D[0]')
+    maya.cmds.setAttr(offset_plus_minus_node + '.input2D[1]', 0.0, 0.0, type='float2')
+    maya.cmds.connectAttr(offset_plus_minus_node + '.output2D', node + '.filmOffset')
```

Comment:
Rather than using "filmOffset" on the camera, I think it's better to use the "pan" attribute, so the connection becomes:
```
mmReprojection.outPan -> offset_pluseMinusAverage1.input2D[1]
offset_pluseMinusAverage1.output2D -> cameraShape.pan
```

The reason for this is because users may want to solve the `filmOffset` values on the camera, or it may already contain some values that need to be maintained.